src/roulette.py

```
@@ -127,6 +130,19 @@ def get(self, bin: int) -> Bin:
         """
         return self.bins[bin]
 
+    def get_Outcome(self, name: str) -> Outcome:
```

This should raise an exception if the string isn’t the name of a known Outcome